examples/e2e/steps/etl/data_loader.py

```diff
@@ -0,0 +1,59 @@
+#  Copyright (c) ZenML GmbH 2023. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+
+from typing import Annotated
+
+import pandas as pd
+from utils.misc import generate_random_data
+
+from zenml import step
+from zenml.client import Client
+from zenml.logger import get_logger
+
+logger = get_logger(__name__)
+
+artifact_store = Client().active_stack.artifact_store
+
+
+@step
+def data_loader(
```

Comment:
For the final example, I think using an actual ML dataset might be more interesting than the random dataset you're using here. E.g., one of the maybe: https://scikit-learn.org/stable/datasets/toy_dataset.html

Reply:
Absolutely, have it on my TODOs. Was just not sure, if the toy dataset is anyhow better than random.
While bringing a real-life case with external DWH overloads this code with the hassle of configuring such a source on all possible clouds (S3, GCP, local, Azure).

If the toy dataset is better from your PoV - I'll rework it, with no arguments.

Reply:
done